src/views/GovernanceReviewTeam/GRBReview/GRBReviewerForm.test.tsx

```diff
@@ -2,36 +2,34 @@ import React from 'react';
 import { MemoryRouter, Route } from 'react-router-dom';
 import { render, screen } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
-
-import { systemIntake } from 'data/mock/systemIntake';
-import { MessageProvider } from 'hooks/useMessage';
-import GetCedarContactsQuery from 'queries/GetCedarContactsQuery';
 import {
-  CreateSystemIntakeGRBReviewerQuery,
-  GetSystemIntakeGRBReviewersQuery,
-  UpdateSystemIntakeGRBReviewerQuery
-} from 'queries/SystemIntakeGRBReviewerQueries';
+  CreateSystemIntakeGRBReviewerDocument,
+  GetSystemIntakeGRBReviewersDocument,
+  SystemIntakeGRBReviewerFragment,
+  SystemIntakeGRBReviewerRole,
+  SystemIntakeGRBReviewerVotingRole,
+  UpdateSystemIntakeGRBReviewerDocument
+} from 'gql/gen/graphql';
 import {
   CreateSystemIntakeGRBReviewer,
   CreateSystemIntakeGRBReviewerVariables
-} from 'queries/types/CreateSystemIntakeGRBReviewer';
-import {
-  GetCedarContacts,
-  GetCedarContactsVariables
-} from 'queries/types/GetCedarContacts';
+} from 'gql/gen/types/CreateSystemIntakeGRBReviewer';
```

Comment:
Moving forward we don't want to import anything from the `gql/gen/types` folders.  These are only being generated here because of the legacy gen tooling.  Once that is deprecated, this generated folder will be removed.  I know it's confusing to have it generated here (we can probably see if we can change the config to not have this generated here as it is not generated by the new tooling).  If you need to import these types they should be imported directly from `gql/gen/graphql`.  To target specific types within the specified query/mutation:

You need to remove:
```
// import {
//   CreateSystemIntakeGRBReviewer,
//   CreateSystemIntakeGRBReviewerVariables
// } from 'gql/gen/types/CreateSystemIntakeGRBReviewer';
```

This is the new method/imports. You also need to identify the parent level `__typename` - mutation/query
```
import {
  CreateSystemIntakeGRBReviewerMutation,
  CreateSystemIntakeGRBReviewerMutationVariables,
} from 'gql/gen/graphql';

const createSystemIntakeGRBReviewerQuery: MockedQuery<
  CreateSystemIntakeGRBReviewerMutation,
  CreateSystemIntakeGRBReviewerMutationVariables
> = {
  request: {
    query: CreateSystemIntakeGRBReviewerDocument,
    variables: {
      input: {
        systemIntakeID: systemIntake.id,
        euaUserId: contact.euaUserId,
        votingRole: SystemIntakeGRBReviewerVotingRole.VOTING,
        grbRole: SystemIntakeGRBReviewerRole.CMCS_REP
      }
    }
  },
  result: {
    data: {
      __typename: 'Mutation',
      createSystemIntakeGRBReviewer: grbReviewer
    }
  }
};
```

This should be applied to all other areas where we have crossed over a new query/mutation

Also to target nested properties of query, we can either import the fragments if they exist, or we have to drill into the parent type to access the nested type.  This more closely couples the generated type with the data from the query.  For example: 

```
type SystemIntakeGRBReviewers = GetSystemIntakeGRBReviewersQuery['systemIntake'];
```

To target a possibly null nested type you can use the `NonNullable`  TS utility type.  Additionally if the nested type is an array, you target that with `[0]` in tyepscript
```
type SystemIntakeGRBReviewers = NonNullable<GetSystemIntakeGRBReviewersQuery['systemIntake']>['grbReviewers'][0];
```

Reply:
I updated the mock queries to use the new type references. Thanks for the examples!

> Also to target nested properties of query, we can either import the fragments if they exist, or we have to drill into the parent type to access the nested type.

Just to clarify, do I need to make a change somewhere related to this or were you just sharing information? I do know how to access properties in an object, but I prefer using fragments if more than one query/mutation uses the same fields and we're going to use the type in multiple places.

Reply:
My understanding is that you'll only need to do those last 2 examples Patrick shared if you _don't_ have a fragment (or don't want to make one just for the sake of accessing an autogenerated type)

I could see it being useful in an instance where a fragment's never re-used (might be a bit confusing to have a fragment just to get codegen to make a type appear), but I also don't really know if that's so bad 🤷